packages/components/src/assay/index.ts

```diff
@@ -21,23 +30,33 @@ import { onAssayRunChange, onAssayDesignChange } from './actions';
 
 export {
     assayPage,
```

Comment:
I believe many of these exported items are no longer needed now that the page components are in ui-components. I did a quick search and I think you can remove the following:
assayPage, getAssayImportNotificationMsg, getAssayRunDeleteMessage, AssayDesignHeaderButtons, AssayRunDetailHeaderButtons, AssayBatchHeaderButtons, AssayHeader, AssayImportDataButton, AssayReimportRunButton, AssayResultDeleteModal, AssayRunDeleteModal, AssayTypeSummary, onAssayRunChange, onAssayDesignChange, UpdateQCStatesButton

Reply:
Whee!  Done.